Create.js

```diff
@@ -0,0 +1,86 @@
+import Store from './Store.js';
```

Comment:
- [ ] As the requirements mentioned, it is necessary for you to include all your JavaScript modules into a `modules` folder, so you can keep all of them organized, so please make sure to implement it.